auth/server/src/main/java/com/seagates3/policy/BucketPolicyAuthorizer.java

```diff
@@ -59,11 +57,11 @@
     // AccessDenied if Deny found
     try {
       String requestedOperation =
-          identifyOperationToAuthorize(requestBody).toLowerCase();
+          identifyOperationToAuthorize(requestBody.get("S3Action"));
```

Comment:
lets try to use constants instead of hardcoded strings

Reply:
fixed